### PR TITLE
Setting butler request size limit for 50mb by default

### PR DESCRIPTION
**Pull request by lucasbfernandes**
Fix: Request entity too large caused by large payloads sent from metacontroller.

### CODE REVIEW

File: install/helm-chart/templates/_helpers.tpl [hunk 1]
```diff
@@ -110,6 +110,8 @@ env:
   value: "http://charlescd-butler.{{ .ChartContext.Release.Namespace }}.svc.cluster.local:3000"
 - name: BUTLER_NAMESPACE
   value: {{ .ChartContext.Release.Namespace }}
+- name: REQUEST_SIZE_LIMIT
+  value: {{ .RangeContext.requestSizeLimit }}
```
Comment (fandrade-zup) [inline install/helm-chart/templates/_helpers.tpl:114]: The right way is passing the value in quotes:

value: {{ .RangeContext.requestSizeLimit | default 50mb| quote}}

File: install/helm-chart/values.yaml [hunk 2]
```diff
@@ -46,6 +46,7 @@ CharlesApplications:
       password: 3f2Yq8R4HhDCnefR
       name: charlescd_butler
       ssl: false
+    requestSizeLimit: '50mb'
```
Comment (fandrade-zup) [inline install/helm-chart/values.yaml:49]: Right way is:
requestSizeLimit: 50mb

**Review by fandrade-zup — CHANGES_REQUESTED** [review]
2 comments on helm chart

File: install/helm-chart/templates/_helpers.tpl (hunk 1 above)
Comment (lucasbfernandes) [reply install/helm-chart/templates/_helpers.tpl]: fixed here https://github.com/ZupIT/charlescd/pull/1228/commits/51713b4c48bdf79823361ef8697f725dfb60ea58

File: install/helm-chart/values.yaml (hunk 2 above)
Comment (lucasbfernandes) [reply install/helm-chart/values.yaml]: fixed here https://github.com/ZupIT/charlescd/pull/1228/commits/51713b4c48bdf79823361ef8697f725dfb60ea58

**Review by fandrade-zup — APPROVED** [review]
LGTM

**Review by thallesfreitaszup — APPROVED** [review]

**Pull request merged**